text/chapter11.md

```diff
@@ -85,19 +85,21 @@ The game is very simple, but the aim of the chapter is to use the `purescript-tr
 
 We will start by looking at some of the monads provided by the `purescript-transformers` package.
 
-The first example is the `State` monad, which provides a way to model _mutable state_ in pure code. We have already seen two approaches to mutable state provided by the `Eff` monad, namely the `REF` and `ST` effects. `State` provides a third alternative, but it is not implemented using the `Eff` monad.
+The first example is the `State` monad, which provides a way to model _mutable state_ in pure code. We have already seen an approach to mutable state provided by the `Effect` monad. `State` provides an alternative.
```

Comment:
Is mutable state with `Effect` covered in ch8? This is mentioned a few times. Might want to strip out all these references.